monarchmoney/monarchmoney.py

```diff
@@ -404,6 +404,59 @@ async def get_transactions(
             operation="GetTransactionsList", graphql_query=query, variables=variables
         )
 
+    async def create_transaction(
+        self,
+        date: str,
+        account_id: str,
+        amount: float,
+        merchant_name: str,
+        category_id: str,
+        notes: str = ""
+    ) -> Dict[str, Any]:
+        """
+        Creates a transaction with the given parameters
+        """
+        query = gql(
+            """
+      mutation Common_CreateTransactionMutation($input: CreateTransactionMutationInput!) {
+        createTransaction(input: $input) {
+          errors {
+            ...PayloadErrorFields
+            __typename
+          }
+          __typename
```

Comment:
Please also return `id` for the newly created transaction.

Reply:
got it!